### ISSUE TITLE
Mobile theme

### ISSUE THREAD

**Issue by dkuep** [guaka/refugeewiki#5]
Often times refugee-related info is required, I'm not at home at my desktop. Can we add a mobile theme for rw?

**Comment by guaka** [comment]
yes, it's high on my wiki todo list I also want this for the other wikis I'm hosting...
I looked into it but no clear solution yet

cc @simison

**Comment by dkuep** [comment]
Aren't there 'themes' including mobile view?

**Comment by guaka** [comment]
I checked the responsive skins, but they were far from great.

See also
https://www.mediawiki.org/wiki/Manual:Mobiles,_tablets_and_responsive_design

**Comment by dkuep** [comment]
Is the theme, mediawiki and wikipedia use, not free?

**Comment by simison** [comment]
https://www.mediawiki.org/wiki/Extension:MobileApp
https://www.mediawiki.org/wiki/Extension:MobileFrontend

From https://www.mediawiki.org/wiki/Category:Extensions_used_on_Wikimedia
